vendor/github.com/bitrise-io/xcode-project/xcodeproj/proj.go

```diff
@@ -55,6 +65,17 @@ func parseProj(id string, objects serialized.Object) (Proj, error) {
 	}, nil
 }
 
+func hasTargetNode(id string, objects serialized.Object) (bool, error) {
+	switch _, err := objects.Object(id); {
+	case err == nil:
+		return true, nil
+	case serialized.IsKeyNotFoundError(err):
+		return false, nil
+	default:
+		return false, err
+	}
```

Comment:
_, err := objects.Object(id)
if err
```suggestion
	_, err := objects.Object(id)
	if err != nil {
	    if serialized.IsKeyNotFoundError(err) {
	        return false, nil
	     }
	     return false, err
	}
        return true, nil
```